docs/02-guides/cache-control.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Cache-Control headers
+description:
+  This guide explains how you can add Cache-Control headers in your routes in
+  Front-Commerce.
+tags:
+  - routes
+  - cache
+---
+
+<SinceVersion tag="3.6" />
+
+<p>{frontMatter.description}</p>
```

Comment:
I think the description bring a nice "placeholder sentence" to introduce the guide, although I understand how it might be considered a bit useless. I'll remove it.